### ISSUE TITLE
Multiline error messages are cut off and don't appear in ALEDetail

### ISSUE THREAD

**Issue by dbatten5** [dense-analysis/ale#3962]
## Information

some error messages, eg. those from `mypy` with `pretty` set to `true` are multi line, however only the top line is shown in the command line and in `ALEDetail`:

![image](https://user-images.githubusercontent.com/17271241/139685546-2282fc2c-a3db-4f10-8809-1793b0f2cb49.png)

![image](https://user-images.githubusercontent.com/17271241/139686026-d3cd0588-d61f-4793-9f23-299bb125a83f.png)

i can see the full error message in `ALEInfo`:

```
xxx.py:2:1: error: Cannot find implementation or library stub for
module named "pydantic"  [import]
    from pydantic import BaseModel
    ^
```

**VIM version**

```
NVIM v0.5.0
Build type: Release
```

Operating System: Mac OSX 10.15.7 Catalina

## What went wrong

as described above. i had a look through the docs and open issues but couldn't find much on this, is there a setting or config option i'm missing?

**Comment by nospam2998** [comment]
I'm unable to reproduce, likely because pedantic probably have library stubs now with version 1.10.18. I tried creating this python file:

```python
from pedantic import BaseModel
```

Running `mypy --pretty .` only says:

> Success: no issues found in 1 source file

Would it be possible to get a code example which reproduces this without any dependencies, please?
